blogposts/announcing-sourcegraph-3.6.md

```diff
@@ -0,0 +1,162 @@
+---
+title: 'Sourcegraph 3.6: Bitbucket Server plugin, search performance improvements, and quick links'
+author: Christina Forney
+publishDate: 2019-07-20T10:00-07:00
+tags: [
+  blog
+]
+slug: sourcegraph-3.6
+heroImage: /sourcegraph-mark.png
+published: true
+---
+
+[Sourcegraph](https://about.sourcegraph.com/) is the standard developer platform for code search and navigation at many of the largest and most exacting technology companies. With Sourcegraph, every company has access to the same kind of tools that Google and Facebook developers use every day.
+
+<div style="padding-left: 2rem">
+
+[**🌎 Sourcegraph plugin provides native code intelligence to Bitbucket Server**](#)<br />
+
+[**📉 30% indexed search latency improvement for small and medium instances**](#)<br />
+
+[**🔗 Add quick links to repositories or important resources below the search bar**](#)<br />
+
+[**☁️ Added support for Bitbucket Cloud external service configuration**](#)<br />
+
+[**🛠 Improved external service configuration and syncing experience for admins**](#)<br />
+
+[**🙅‍♀️ Easily exclude GitHub repositories from syncing (by regexp)**](#)<br />
+
+[**✈️ Meet us in person at GopherCon!**](#)<br />
+
+[**📝 Changelog**](#35-changelog)<br />
+
+[**🎖️ Thank you**](#thank-you)<br />
+
+</div>
+
+**Deploy or upgrade:** [Local](https://docs.sourcegraph.com/#quickstart-guide) | [AWS](https://github.com/sourcegraph/deploy-sourcegraph-aws) | [DigitalOcean](https://marketplace.digitalocean.com/apps/sourcegraph?action=deploy&refcode=48dfb3ccb51c) | [Kubernetes cluster](https://github.com/sourcegraph/deploy-sourcegraph)
+
+## Sourcegraph plugin provides native code intelligence to Bitbucket Server
+
+TO DO: VIDEO
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+Admins can now install the [Sourcegraph for Bitbucket Server plugin](https://github.com/sourcegraph/bitbucket-server-plugin) on their Bitbucket Server instance to provide code intelligence automatically to their teams. Code intelligence, previously available only through the browser extension, includes hovers, tooltips, and line decorations while browsing and reviewing code in Bitbucket Server.
+
+Previously, many users were not aware of the Sourcegraph browser extension. Now, admins can provide an easier onboarding path for all team members who want to browse or review code on Bitbucket Server.
```

Comment:
```suggestion
```

In this case, I don't think it's worth stating the problem as most uses didn't know they were missing anything.